### PR TITLE
🌐 [i18n-KO] Translated `custom_tools.mdx` to Korean 

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `custom_tools.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)
@sgugger, @ArthurZucker, @eunseojo 
May you please review this PR?

### CODE REVIEW

**Comment by jungnerd** [comment]
LGTM👍

**Comment by amyeroberts** [comment]
Thanks for the PR, @sim-so! 

There's since been an update on main to all of our documentation files. Could you update the extension from `.mdx` to `.md` to match please?

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
It's a new type of documentation, but you translated it perfectly! 
Thanks for the good translation 😀
Please check out the review below!

File: docs/source/ko/custom_tools.mdx [hunk 1]
```diff
@@ -0,0 +1,748 @@
...
+=====
+
+[...]
+````
+
+`run` 프롬프트의 예와는 반대로, 각 `chat` 프롬프트의 예에는 *Human(사람)*과 *Assistant(어시스턴트)* 간에 하나 이상의 교환이 있습니다. 모든 교환은 `run` 프롬프트의 예와 유사한 구조로 되어 있습니다.
+사용자의 입력이 *Human:* 뒤에 추가되며, 에이전트에게 코드를 생성하기 전에 수행해야 할 작업을 먼저 생성하라는 메시지가 표시됩니다. 
+교환은 이전 교환을 기반으로 할 수 있으므로 위와 같이 사용자가 "**이** 코드를 시도했습니다"라고 입력하면 이전에 생성된 에이전트의 코드를 참조하여 과거 교환을 참조할 수 있습니다.
+
+`.chat`을 실행하면 사용자의 입력 또는 *task*가 미완성된 양식의 예시로 캐스팅됩니다:
```
[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/custom_tools.mdx:211]: > task가 프롬프트에 있는 "Task: "를 해석될 수도 있어서 참고 부탁드립니다!
비슷한 고민을 하다 후자로 이해하고 `task`를 그대로 썼었는데요, 이 문장에서는 번역한 단어가 더 자연스러운 것 같아요. 수정 제안 감사합니다!

**Comment by sim-so** [comment]
Thanks for letting me know about the change! @amyeroberts! 
I updated the extention to `.md`.

Could you review this PR?
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**